src/jats/exporter/jats-exporter.ts

```diff
@@ -1814,6 +1847,26 @@ export class JATSExporter {
     }
     body.removeChild(container)
   }
+  private moveAwards = (front: HTMLElement, body: HTMLElement) => {
```

Comment:
Are we expecting funding groups in the body?

Reply:
[ buildBody ](https://github.com/Atypon-OpenSource/manuscripts-transform/blob/f7b4538f61dff35b4c5121d21965dd6d07dd33c1/src/jats/exporter/jats-exporter.ts#L540) takes the entire manuscriptNode and serializes it, that's why front nodes are not serialized directly using the Serializer. 
I wanted to use the serializer that's why I had to move them out.

Reply:
Ah, I didn't notice this is the exporter. This is another thing we might want to look at later.